mod_semantic_map/SemMap map
string                  object
geometry_msgs/Point     location
---